dozer/cogs/teams.py

```diff
@@ -61,6 +64,49 @@ async def teamsfor(self, ctx, user: discord.Member = None):
     `{prefix}teamsfor member` - Returns all team associations with the mentioned user. Assumes caller if blank.
     """
 
+    @command()
+    @guild_only()
+    async def compcheck(self, ctx, event_type: str, event_key):
+        """Allows you to see people in the Discord server that are going to a certain competition."""
+        if event_type.lower() == "frc":
+            teams_raw = await TBA(ctx.bot).session.event_teams(event_key)
+            teams = [team.team_number for team in teams_raw]
+        elif event_type.lower() == "ftc":
+            teams_raw = json.loads(await TOA(ctx.bot).parser.req(f"/api/event/{event_key}/teams"))
+            teams = [team['team']['team_number'] for team in teams_raw]
+        else:
+            raise BadArgument("Unknown event type!")
+        found_mems = False
+        e = discord.Embed(type='rich')
+        e.title = 'Members going to {}'.format(event_key)
+        for team in teams:
+            members = await TeamNumbers.get_by(team_type=event_type.lower(), team_number=team)
+            memstr = ""
+            for member in members:
+                mem = ctx.guild.get_member(member.user_id)
+                if mem is not None:
+                    newmemstr = "{} {} \n".format(mem.display_name, mem.mention)
+                    if len(newmemstr + memstr) > 1023:
+                        e.add_field(name=f"Team {team}", value=memstr)
+                        memstr = ""
+                    memstr += newmemstr
+                    found_mems = True
+            if len(memstr) > 0:
+                if len(e.fields) == 25:
+                    await ctx.send(embed=e)
+                    e = discord.Embed(type='rich')
+                    e.title = 'Members going to {}'.format(event_key)
```

Comment:
This is likely to cause spam when used with i.e. a championship division or DCMP. This would be a good place for pagination.

Reply:
I don't know how to paginate